app/views/content/train-to-be-a-teacher/who-do-you-want-to-teach.md

```diff
@@ -3,114 +3,85 @@ title: Who do you want to teach
 
-To start early years teacher training, you’ll need these qualifications (or to demonstrate an equivalent standard):
+$get-school-experience$
 
-- maths grade 4 (C)
-- English grade 4 (C)
-- science grade 4 (C)
+If you’re an undergraduate and interested in teaching chemistry, computing, languages, maths or physics then you could do a [paid teaching internship](/teaching-internship-providers) to find out if teaching is right for you.
 
-If you do postgraduate early years teacher training, you’ll need a 2:2 bachelor’s degree with honours or higher.
+### Pupils with special educational needs
+Most disabled pupils and pupils with special educational needs learn in mainstream schools. Wherever you train and work – whether it’s mainstream schools, special schools, or both – you’ll work with pupils with a range of needs.
 
-[Contact accredited early years training providers to apply for a place](https://www.gov.uk/government/publications/early-years-initial-teacher-training-itt-providers-and-school-direct-early-years-lead-organisations/early-years-initial-teacher-training-itt-providers-and-school-direct-lead-organisations).
+You can choose to apply for a teacher training course that specialises in SEND or you can gain the skills and experience as you train and teach.
 
-[Learn about funding](https://www.gov.uk/guidance/early-years-initial-teacher-training-2023-to-2024-funding-guidance).
+Learn more about the [routes to teach pupils with special educational needs and disabilities (SEND)](/teach-disabled-pupils-and-pupils-with-special-educational-needs).
 
-### Undergraduate early years teacher training
+## Teaching other stages
 
-This is a full-time bachelor’s degree in an early childhood-related subject leading to EYTS. You will usually study for 3 or 4 years.
+### Nursery or pre-school
+If you want to help young children under 5 years old to develop, you might consider becoming an early years teacher.
 
-Fees for full-time courses are usually around £9,250 but you can:
+Early years teachers play an essential role in children’s development up to the age of 5 and provide high-quality early education.
 
-- get [funding that you do not have to pay back](/funding-and-support/scholarships-and-bursaries) for certain subjects
-- apply for a [student loan](/funding-and-support/tuition-fee-and-maintenance-loans)
+Learn more about [the routes to become an early years teacher](/early-years-teaching-training).
 
-### Graduate early years teacher training
+### Further education
+If you’ve got industry experience and a passion for teaching young people over 16 and adults, then further education could be for you. 
 
-This is a full-time post-graduate course leading to EYTS. You will usually study for 1 academic year.
+There are three key areas, vocational courses covering areas like plumbing and health care, academic courses on subjects like English and maths, and basic skills courses covering literacy and numeracy. 
```

Comment:
Does this maybe need a colon and bullets? e.g.

There are 3 key areas:

- vocational etc.
- academic etc.
- basic skills etc.